x-pack/platform/plugins/shared/streams/server/lib/streams/assets/asset_client.ts

```diff
@@ -277,6 +288,31 @@ export class AssetClient {
     );
   }
 
+  async findQueries(name: string, query: string) {
+    const assetsResponse = await this.clients.storageClient.search({
+      size: 10_000,
+      track_total_hits: false,
+      runtime_mappings: {
+        [QUERY_FEATURE_NAME]: { type: 'keyword' },
+        [QUERY_FEATURE_FILTER]: { type: 'keyword' },
+      },
```

Comment:
I think this is fine for the expected size of the stream, let me know if you have concerns. The alternative is to update the schema and do a migration